operator/src/test/java/oracle/kubernetes/operator/helpers/AdminPodHelperTest.java

```diff
@@ -152,6 +164,41 @@ public void whenExistingAdminPodHasBadVersion_replaceIt() {
         pod -> pod.getMetadata().putLabelsItem(RESOURCE_VERSION_LABEL, "??"));
   }
 
+  @Test
+  public void whenExistingManagedPodHasUnknownCustomerLabel_designateForRoll() {
```

Comment:
Need to test the case a label is removed?

Reply:
it should be the same, but I will see if there is an easy way to mutate the model rather than the existing pod in the test